docs/analytics-ga4/ga4-finder-tracker.md

```diff
@@ -0,0 +1,122 @@
+# Google Analytics 4 finder tracker
+
+This script is used to add GA4 tracking to the GOVUK finders in `finder-frontend` such as https://www.gov.uk/search/all. These finders use client side JavaScript to update their results in response to a change in the keyword or filters. Therefore, this tracker hooks in to the JavaScript `change` event that fires whenever a user updates their search. It then does some processing to push an appropriate GA4 event depending on what change was made to their search. This differs from the `ga4-form-tracker`, Firstly, because our finders are all contained in one large form. The `ga4-form-tracker` only allows for one GA4 event per `<form>` when we need multiple for each element that can change the search results. Secondly, the `ga4-form-tracker` relies on a JavaScript `submit` event. Our finder pages use a `change` event to update their results.
```

Comment:
Few things/nitpicks:

- In the second sentence, how would you feel about changing "keyword" to "search term"?
- How would you feel about changing "It then does some processing to push an appropriate GA4 event... " to "A GA4 schema is then built... "

The above points are very much personal preference things so happy to leave things as they are if you'd prefer!

- Typo: "This differs from the `ga4-form-tracker`, Firstly... "

Reply:
Quick question: how would you feel about changing "GA4 event" to "GTM object"? Only because we refer to GTM objects later in this documentation (and elsewhere in the wider documentation) and I'm wondering if we should keep things consistent. This is more nitpicking though and happy to discuss 👍